Tests/IntegrationTests/Fixtures/1Dimension/selectBoxes.e2e.js

```diff
@@ -23,6 +23,7 @@ test('SelectBox opens below and breaks out of the creation dialog if there\'s en
 
 test('SelectBox opens above in creation dialog if there\'s not enough space below.', async t => {
     await t
+        .resizeWindow(1200, 768)
```

Comment:
Odd, this is not fixed in 8.2 https://github.com/neos/neos-ui/blob/07668ea8fc64784b447601d1fd611570178eaa5a/Tests/IntegrationTests/Fixtures/1Dimension/selectBoxes.e2e.js#L24 

but will be upmerged - and if it helps ...

Reply:
Yes I also wanted to fix it for 8.2 but as we upmerge it I wouldn't create a separate PR. And it seems only I have those resolution issues with the window being to high.

Reply:
And in general we should have a defined window size if it affects the tests, but the global parameter (which I only found on Stackoverflow) for that in test cafe didn't work for me.

Reply:
okay thanks fine with me ;)